pkg/cmd/pipeline/create.go

```diff
@@ -0,0 +1,127 @@
+package pipeline
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/buildkite/cli/v3/internal/io"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/buildkite/go-buildkite/v3/buildkite"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdPipelineCreate(f *factory.Factory) *cobra.Command {
+
+	cmd := cobra.Command{
+		DisableFlagsInUseLine: true,
+		Use:                   "create",
+		Short:                 "Creates a new pipeline",
+		Args:                  cobra.MaximumNArgs(1),
```

Comment:
This doesn't actually seem to take in an any args?
```suggestion
		Args:                  cobra.NoArgs,
```

Reply:
Oh yep.. I initially implemented this to accept a pipeline name but forgot to remove that after asking for inputs